player/lua/osc.lua

```diff
@@ -2468,6 +2483,11 @@ mp.observe_property("fullscreen", "bool",
         request_init()
     end
 )
+mp.observe_property("border", "bool",
+    function(name, val)
+        state.border = val
```

Comment:
I don't see where `state.border` is used. Was it supposed to be a cached value which `window_controls_enabled()` checks instead of using `mp.get_property(...)` on each call?